src/breadcrumb-group/__integ__/breadcrumb-group.test.ts

```diff
@@ -31,12 +31,12 @@ class BreadcrumbGroupPage extends BasePageObject {
     return this.click(dropdownWrapper.findItems().get(index).toSelector());
   }
 }
-const setupTest = (testFn: (page: BreadcrumbGroupPage) => Promise<void>) => {
+const setupTest = (testFn: (page: BreadcrumbGroupPage, browser: WebdriverIO.Browser) => Promise<void>) => {
```

Comment:
Is this second parameter necessary? The `browser` used in line 39 comes from line 35, right?

Reply:
Yes - The browser object in Line 39 comes from line 35. Line 34 is the type definition of testFn (Line 39).